src/components/views/elements/PollCreateDialog.tsx

```diff
@@ -185,18 +191,21 @@ export default class PollCreateDialog extends ScrollableBaseModal<IProps, IState
         return <div className="mx_PollCreateDialog">
             <h2>{ _t("What is your poll question or topic?") }</h2>
             <Field
+                id='poll-topic'
                 value={this.state.question}
                 maxLength={MAX_QUESTION_LENGTH}
                 label={_t("Question or topic")}
                 placeholder={_t("Write something...")}
                 onChange={this.onQuestionChange}
                 usePlaceholderAsHint={true}
                 disabled={this.state.busy}
+                autoFocus={this.state.autoFocusTarget === FocusTarget.Topic}
             />
             <h2>{ _t("Create options") }</h2>
             {
                 this.state.options.map((op, i) => <div key={`option_${i}`} className="mx_PollCreateDialog_option">
                     <Field
+                        id={`option_${i}`}
```

Comment:
Using the autogenerated field ids means the tests rely on running order of tests, add these to have stable ids

Reply:
ID should be scoped to this component, so it doesn't clash